metrics/expvar/expvar.go

```diff
@@ -75,9 +81,35 @@ type histogram struct {
 	hist *hdrhistogram.WindowedHistogram
 
 	name   string
+	count  metrics.Counter
 	gauges map[int]metrics.Gauge
 }
 
+// Name returns the name of the histogram
+func (h *histogram) Name() string {
+	return h.name
+}
```

Comment:
I'm researching and making a case for histogram distributions being useful rather than just a few percentiles. Not sure how to generalize logging a histogram into Metrics, so reading an exported Distribution (see the updated hdr.go) directly is required so it can be encoded as needed. I used the String mechanism as a workaround for now.